ivy/functional/backends/paddle/experimental/elementwise.py

```diff
@@ -122,17 +122,17 @@ def nansum(
     keepdims: Optional[bool] = False,
     out: Optional[paddle.Tensor] = None,
 ) -> paddle.Tensor:
-    raise IvyNotImplementedException()
+    return paddle.nansum(x)
 
 
 def gcd(
-    x1: Union[paddle.Tensor, int, list, tuple],
-    x2: Union[paddle.Tensor, float, list, tuple],
```

Comment:
x2 should also contain int dtype

Reply:
please discard this change as the function declaration is universal throughout all backends, we can only modify function body